src/ripple/rpc/handlers/LedgerData.cpp

```diff
@@ -50,11 +50,12 @@ Json::Value doLedgerData (RPC::Context& context)
     if (!lpLedger)
         return jvResult;
 
-    uint256 resumePoint;
+    boost::optional<ReadView::key_type> key;
+    key.emplace();
```

Comment:
What does boost::optional buy here? Oh, I see it's what's returned by that little ReadView `succ`ker. `emplace` with no args creates via default constructor a zeroed out uint256 key?